RELEASE/relay/relay_autoscend.ash

```diff
@@ -312,10 +314,12 @@ void main()
 	writeln("Tavern: " + get_property("tavernLayout") + "<br>");
 	if(my_class() == $class[Ed])
 	{
-		writeln("Combats: " + get_property("sl_edCombatCount") + "<br>");
-		writeln("Combat Rounds: " + get_property("sl_edCombatRoundCount") + "<br>");
+		writeln("Combats: " + get_property("auto_edCombatCount") + "<br>");
+		writeln("Combat Rounds: " + get_property("auto_edCombatRoundCount") + "<br>");
 	}
-	writeln("Version (sl_ascend): " + svn_info("soolar-sl_ascend-trunk-RELEASE").last_changed_rev + "<br>");
+
+	//TODO: need way to track version independent of svn branch since you can have different branches checked out
+	writeln("Version (autoscend): " autoscend_current_version() + "<br>");
```

Comment:
I should remove that todo, but i needed a way to track version independent of branch if we are going to use the beta branch rather than feature flagging things behind the beta property.

Reply:
You can get the checkout URL of a script with `svn_info("scriptname").url.

```
> ash svn_info("sl_ascend")

Returned:     record {string url; int revision; string last_changed_author; int     last_changed_rev; string last_changed_date;}
url =>     https://github.com/soolar/sl_ascend/trunk/RELEASE
revision => 1004
last_changed_author     => soolar
last_changed_rev => 1004
last_changed_date =>     2019-09-07 22:33:51 EDT (Sat, 07 Sep 2019)
```

Reply:
You can get the checkout URL of a script with `svn_info("scriptname").url. I don't know how to make git fake an svn repo for a non-master branch, but if you can figure that out there might be a fix as simple as checking if "beta" appears in that url.

```
> ash svn_info("sl_ascend")

Returned:     record {string url; int revision; string last_changed_author; int     last_changed_rev; string last_changed_date;}
url =>     https://github.com/soolar/sl_ascend/trunk/RELEASE
revision => 1004
last_changed_author     => soolar
last_changed_rev => 1004
last_changed_date =>     2019-09-07 22:33:51 EDT (Sat, 07 Sep 2019)
```